config.docker.toml

```diff
@@ -60,6 +63,13 @@ max_connections = 40000
 #cert_file = "certs/test.crt"
 #key_file = "certs/test.key"
 
+[profile]
+enable = true
+
+[profile.listener]
+addr = ":8082"
+max_connections = 100
```

Comment:
It's an optional section; I left a [comment](https://github.com/mozilla-services/pushgo/pull/203/files#r23577343) clarifying.